### ISSUE TITLE
WOLF Transport computer erases route payload value after decoupling with other parts of the ship

### ISSUE THREAD

**Issue by diegogalati** [UmbraSpaceIndustries/MKS#1546]
KSP 1.12.2
MKS v112.0.1-bleeding-edge.1

Start with a ship containing a

- a Wolf transport computer
- a Wolf cargo container
- Any other part/group of parts coupled through a decoupler with rest of the ship

When try to create a route with transport computer, the part action window report the correct route payload (eg. 15)

After decoupling with other parts through decoupler, the transport computer changes the route payload to 0 (it was 15)

This doesn't allow to complete the route

Similar to the issue #1544

**Comment by RookFett** [comment]
Are you disconnecting your wolf cargo from the main ship completely?   Does it have an SAS probe on the container or the final stack configuration?  Is the transport computer on the container?

In testing - if the Wolf Cargo container or the SAS probe controlling the container is not the "root" part of the stack, it will break like you state.   

I used the stock GDLV3 rocket to test.   I made the SAS on top of the Wolf Cargo root.   This rocket has two SAS probe cores - one on the final stage, then the one on the cargo.

I switch from the cargo to the SAS probe (on cargo) as root, and it worked as expected.  As soon as I made something other than those two root - it would break the route.

So - try making the cargo pod or the SAS probe on your final stack as root and see if you see the same problem.

Remember - the cargo needs a probe core to actually connect it to the depot - so if you build a launch vehicle, then add the cargo pod/probe - it must likely won't be root - breaking when you disconnect the pod from the stack.
